src/app.controller.ts

```diff
@@ -2,8 +2,8 @@ import {Get, Controller} from '@nestjs/common';
 
 @Controller()
 export class AppController {
-  @Get('/server')
+  @Get("/server")
   server(): null {
-    return;
+    return
```

Comment:
This was updated from "prettier" for formatting.

Reply:
This was my mistake, I originally committed without prettier running. I have added prettier as part of the CI in a separate PR to try to avoid this in the future.